develop/webservices/WS_v1/WS_CustomDataTypes_v1/DMAParameterInfo.md

```diff
@@ -6,21 +6,55 @@ uid: DMAParameterInfo
 
 | Item | Format | Description |
 |--|--|--|
-| ID | Integer | The ID of the parameter. |
-| ParameterName | String | The name of the parameter. |
-| InfoSubText | String | The parameter description. |
-| IsMonitored | Boolean | Whether the value of the parameter is being monitored by an alarm template. |
-| IsNumerical | Boolean | Whether the value of the parameter is numerical. |
-| HasRange | Boolean | Determines whether the value of the parameter is defined within a certain range. |
-| RangeLow | Double | If *HasRange* is true, this determines the lowest value of the parameter range. |
-| RangeHigh | Double | If *HasRange* is true, this determines the highest value of the parameter range. |
-| Options | String | The parameter options (extra flags to e.g. indicate special formatting instructions). |
-| Unit | String | The unit of measure of the parameter. |
-| PrimaryKeyID | Integer | If the parameter is a table parameter, the primary key. |
-| DisplayKeyID | Integer | If the parameter is a table parameter, the display key. |
-| IsLogarithmic | Boolean | Indicates whether the trend data of this parameter should be displayed on a logarithmic scale. |
-| IsTableColumn | Boolean | Whether or not the parameter is a table column parameter. |
-| IsTable | Boolean | Whether or not the parameter is a table parameter. |
+| ID               | Integer | The ID of the parameter. |
+| ParameterName    | String | The name of the parameter. |
+| InfoSubText      | String | The parameter description. |
+| IsMonitored      | Boolean | Whether the value of the parameter is being monitored by an alarm template. |
+| IsNumerical      | Boolean | Whether the value of the parameter is numerical. |
+| HasRange         | Boolean | Determines whether the value of the parameter is defined within a certain range. |
+| RangeLow         | Double | If *HasRange* is true, this determines the lowest value of the parameter range. |
+| RangeHigh        | Double | If *HasRange* is true, this determines the highest value of the parameter range. |
+| RangeLowDisplay  | String |  |
```

Comment:
@wimme , can you check these two?

Reply:
RangeLowDisplay and RangeHighDisplay contain the display value (as defined in the protocol) for the raw values of RangeLow and RangeHigh.